.gitignore

```diff
@@ -103,3 +103,7 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# .idea
+.idea
```

Comment:
Change this to `.idea/` and it should ignore all `idea` directories.
Currently, there are still some remaining.